labs/lab4/chapter4.md

```diff
@@ -218,7 +217,7 @@ $ oc delete pod/mariadb pod/wordpress
 
 Verify they are terminating or are gone:
 ```bash
-$ oc get pods -w
```

Comment:
if we don't add the `-w` then we don't need the **NOTE** below

Reply:
removed the note